core/summary_api.php

```diff
@@ -116,126 +199,76 @@ function summary_print_by_enum( $p_enum ) {
 				ORDER BY ' . $p_enum . ' ' . $t_status_query;
 	$t_result = db_query( $t_query );
 
-	$t_last_value = -1;
+	$t_cache = array();
+	$t_bugs_total_count = 0;
 	$t_bugs_open = 0;
 	$t_bugs_resolved = 0;
 	$t_bugs_closed = 0;
 	$t_bugs_total = 0;
```

Comment:
The above 4 assignments are unnecessary, the variables' values are overwritten in the foreach loop